From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Tue, 25 Feb 2014 11:02:26 +0200
Subject: [PATCH] Add changelog

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>

diff --git a/gbp-test.spec b/gbp-test.spec
index 0a8fecc..9f0e198 100644
--- a/gbp-test.spec
+++ b/gbp-test.spec
@@ -44,3 +44,10 @@ install %{SOURCE0} %{buildroot}/%{_datadir}/%{name}
 %defattr(-,root,root,-)
 %dir %{_datadir}/%{name}
 %{_datadir}/%{name}
+
+
+
+%changelog
+* Tue Feb 25 2014 Markus Lehtonen <markus.lehtonen@linux.intel.com> 1.1-2
+- Version 1.1
+
