From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Thu, 17 Jul 2014 11:27:27 +0300
Subject: [PATCH] Auto-import file(s) from branch 'srcdata/gbp-test/master':   
  .gbp.conf

Gbp: Ignore
Gbp-Rpm: Ignore

diff --git a/.gbp.conf b/.gbp.conf
new file mode 100644
index 0000000..598b658
--- /dev/null
+++ b/.gbp.conf
@@ -0,0 +1,7 @@
+[DEFAULT]
+vendor = Test-Distro
+packaging-branch = master
+packaging-tag = release/%(version)s
+upstream-branch = upstream
+upstream-tag = upstream/%(upstreamversion)s
+packaging-tag-msg = %(vendor)s release %(version)s
