From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Thu, 8 Nov 2012 19:37:36 +0200
Subject: [PATCH] Create a forked version.

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>

diff --git a/.gbp.conf b/.gbp.conf
index 598b658..1f3671f 100644
--- a/.gbp.conf
+++ b/.gbp.conf
@@ -1,7 +1,7 @@
 [DEFAULT]
 vendor = Test-Distro
-packaging-branch = master
-packaging-tag = release/%(version)s
+packaging-branch = fork
+packaging-tag = fork/release/%(version)s
 upstream-branch = upstream
 upstream-tag = upstream/%(upstreamversion)s
 packaging-tag-msg = %(vendor)s release %(version)s
diff --git a/gbp-test.spec b/gbp-test.spec
index c46a734..4759934 100644
--- a/gbp-test.spec
+++ b/gbp-test.spec
@@ -10,11 +10,11 @@ Source20:   bar.tar.gz
 # Gbp-Ignore-Patches: 0
 Patch0:     my.patch
 Patch10:    my2.patch
-Patch20:    my3.patch
 
 
 %description
 Package for testing the RPM functionality of git-buildpackage.
+Other version.
 
 
 %prep
diff --git a/my3.patch b/my3.patch
deleted file mode 100644
index 9fee859..0000000
--- a/my3.patch
+++ /dev/null
@@ -1,7 +0,0 @@
-diff --git a/README b/README
-index a1311cb..a59f1b9 100644
---- a/README
-+++ b/README
-@@ -1 +1 @@
--Just for testing git-buildpackage.
-+Just for testing GBP.
