From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Tue, 27 Aug 2013 16:57:44 +0300
Subject: [PATCH] Update gbp.conf to rename the generated source package

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>

diff --git a/.gbp.conf b/.gbp.conf
index 1f3671f..d2302db 100644
--- a/.gbp.conf
+++ b/.gbp.conf
@@ -5,3 +5,4 @@ packaging-tag = fork/release/%(version)s
 upstream-branch = upstream
 upstream-tag = upstream/%(upstreamversion)s
 packaging-tag-msg = %(vendor)s release %(version)s
+postbuild = for _r in $GBP_BUILD_DIR/SRPMS/*rpm; do _b=`basename $_r`; _n=`echo $_b | sed s"/src.rpm/other.src.rpm/"`; mv $_r `dirname $_r`/$_n; done
