From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Thu, 8 Nov 2012 19:37:36 +0200
Subject: [PATCH] Version bump to 1.1

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>

diff --git a/gbp-test.spec b/gbp-test.spec
index c46a734..0898d72 100644
--- a/gbp-test.spec
+++ b/gbp-test.spec
@@ -1,6 +1,6 @@
 Name:       gbp-test
 Summary:    Test package for git-buildpackage
-Version:    1.0
+Version:    1.1
 Release:    1
 Group:      Development/Libraries
 License:    GPLv2
