From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Tue, 13 Nov 2012 13:57:35 +0200
Subject: [PATCH] Remove imported patches

And enable automatic patch-generation.

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>

diff --git a/.gbp.conf b/.gbp.conf
index 6c8e847..52a1267 100644
--- a/.gbp.conf
+++ b/.gbp.conf
@@ -6,6 +6,7 @@ upstream-branch = upstream
 upstream-tag = upstream/%(upstreamversion)s
 packaging-tag-msg = %(vendor)s release %(version)s
 packaging-dir = packaging
+patch-export = True
 patch-ignore-path = ^(packaging/.*|\.gbp.conf)
 spec-file = packaging/gbp-test2.spec
 
diff --git a/packaging/gbp-test2.spec b/packaging/gbp-test2.spec
index 95c3dc2..f41abc3 100644
--- a/packaging/gbp-test2.spec
+++ b/packaging/gbp-test2.spec
@@ -2,7 +2,7 @@ Name:       gbp-test2
 Summary:    Test package 2 for git-buildpackage
 Epoch:      1
 Version:    2.0
-Release:    0
+Release:    1
 Group:      Development/Libraries
 License:    GPLv2
 Source10:   ftp://ftp.host.com/%{name}-%{version}.tar.gz
@@ -11,8 +11,6 @@ Source20:   bar.tar.gz
 Source9999: gbp-test2-alt.spec
 # Gbp-Ignore-Patches: 0
 Patch0:     my.patch
-Patch10:    http://example.com/patches/my2.patch
-Patch20:    my3.patch
 Packager:   Markus Lehtonen <markus.lehtonen@linux.intel.com>
 
 %description
@@ -25,7 +23,6 @@ git branch.
 %setup -T -n %{name}-%{version} -c -a 10
 
 %patch
-%patch -P 10 -p1
 
 echo "Do things"
 
diff --git a/packaging/my2.patch b/packaging/my2.patch
deleted file mode 100644
index 5554d9c..0000000
--- a/packaging/my2.patch
+++ /dev/null
@@ -1,16 +0,0 @@
-From fb613432f15bc199a724741883b23fff9946b567 Mon Sep 17 00:00:00 2001
-From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
-Date: Tue, 13 Nov 2012 13:55:46 +0200
-Subject: [PATCH] My modification
-
-Add new file 'mydir/myfile.txt'.
-
-Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>
-
-diff --git a/mydir/myfile.txt b/mydir/myfile.txt
-new file mode 100644
-index 0000000..2cdad29
---- /dev/null
-+++ b/mydir/myfile.txt
-@@ -0,0 +1 @@
-+Dummy
diff --git a/packaging/my3.patch b/packaging/my3.patch
deleted file mode 100644
index 9fee859..0000000
--- a/packaging/my3.patch
+++ /dev/null
@@ -1,7 +0,0 @@
-diff --git a/README b/README
-index a1311cb..a59f1b9 100644
---- a/README
-+++ b/README
-@@ -1 +1 @@
--Just for testing git-buildpackage.
-+Just for testing GBP.
