From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Thu, 8 Nov 2012 14:33:51 +0200
Subject: [PATCH] Add packaging files

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>

diff --git a/packaging/gbp-test-native.spec b/packaging/gbp-test-native.spec
new file mode 100644
index 0000000..38b07e4
--- /dev/null
+++ b/packaging/gbp-test-native.spec
@@ -0,0 +1,34 @@
+Name:       gbp-test-native
+Summary:    Test package for git-buildpackage
+Version:    1.0
+Release:    1
+Group:      Development/Libraries
+License:    GPLv2
+Source1:    %{name}-%{version}.zip
+BuildRequires:  unzip
+
+%description
+Package for testing the RPM functionality of git-buildpackage.
+Mimics a "native" package
+
+
+%prep
+unzip %{SOURCE1}
+%setup -T -D
+
+
+%build
+make
+
+
+%install
+rm -rf %{buildroot}
+mkdir -p %{buildroot}/%{_datadir}/%{name}
+cp -R * %{buildroot}/%{_datadir}/%{name}
+
+
+
+%files
+%defattr(-,root,root,-)
+%dir %{_datadir}/%{name}
+%{_datadir}/%{name}
