From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Tue, 25 Feb 2014 10:57:25 +0200
Subject: [PATCH] Add changelog file

Add an "OBS-style" separate changelog file.

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>

diff --git a/packaging/gbp-test-native.changes b/packaging/gbp-test-native.changes
new file mode 100644
index 0000000..e9886a4
--- /dev/null
+++ b/packaging/gbp-test-native.changes
@@ -0,0 +1,3 @@
+* Tue Feb 25 2014 Markus Lehtonen <markus.lehtonen@linux.intel.com> 1.0-1
+- Initial packaging
+
