From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Thu, 8 Nov 2012 19:49:51 +0200
Subject: [PATCH] Add gbp.conf

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>

diff --git a/.gbp.conf b/.gbp.conf
new file mode 100644
index 0000000..13aa70c
--- /dev/null
+++ b/.gbp.conf
@@ -0,0 +1,5 @@
+[DEFAULT]
+vendor = Test-Distro
+packaging-branch = master
+packaging-tag = release/%(version)s
+packaging-tag-msg = %(vendor)s release %(version)s
